test/quantization/test_quantize_script.py

```diff
@@ -2285,7 +2285,6 @@ def __init__(self):
                 self.adaptive_avg_pool1d = torch.nn.AdaptiveAvgPool1d((1))
                 self.adaptive_avg_pool2d = torch.nn.AdaptiveAvgPool2d((1, 1))
                 self.adaptive_avg_pool3d = torch.nn.AdaptiveAvgPool3d((1, 1, 1))
-                self.elu = torch.nn.ELU()
```

Comment:
I understand why we are removing the module here, but dont see where we test that graph mode quantization works correctly for elu (i.e observers are inserted ).

Reply:
it's added back in the correct way further up the PR stack.  Since this functionality is not released yet, I split the diffs for easier review.